internal/cli-flags/Parser.ts

```
@@ -375,12 +371,6 @@ export default class Parser<T> {
 			consumer.markUsedProperty(key);
 		}
 
-		for (const shorthandName of this.shorthandFlags) {
-			consumer.get(shorthandName).unexpected(
-				descriptions.FLAGS.UNSUPPORTED_SHORTHANDS,
-			);
-		}
```

It felt weird to continue to explicitly raise a diagnostic when shorthand commands are encountered. In my mind, the `--help` output shows what commands are supported, which makes it clear that no shorthands exist. Passing unsupported shorthand commands (which is anything other than `-h`) should fail silently just like passing unsupported full flags.

---

Yeah that makes sense to me. Want to remove the `UNSUPPORTED_SHORTHANDS` diagnostic description too?